templates/taxi/car_detail.html

```diff
@@ -2,14 +2,29 @@
 
 {% block content %}
   <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
+  {{ car.model }}
+  {% if request.user in drivers %}
+    <a href="{% url 'taxi:toggle_driver' pk=car.pk %}" class="btn btn-danger link-to-page"
+   onclick="event.preventDefault(); document.getElementById('toggle-driver-form').submit();">
+    Delete me from this car
     </a>
 
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
+    <form id="toggle-driver-form" action="{% url 'taxi:toggle_driver' pk=car.pk %}" method="POST" style="display:none;">
+      {% csrf_token %}
+      <input type="hidden" name="action" value="remove">
+    </form>
+
+  {% else %}
+    <a href="{% url 'taxi:toggle_driver' pk=car.pk %}" class="btn btn-success link-to-page"
+   onclick="event.preventDefault(); document.getElementById('toggle-driver-form').submit();">
+    Assign me to this car
     </a>
+
+    <form id="toggle-driver-form" action="{% url 'taxi:toggle_driver' pk=car.pk %}" method="POST" style="display:none;">
+      {% csrf_token %}
+      <input type="hidden" name="action" value="add">
+    </form>
```

Comment:
why did you decide to make it as form ?

Reply:
Csrf? And in addition I guess we could want to add any further logic, like checking if car exists, or anything else, just in case. But since I only needed to provide certain functioning, I only made model and chose this method.